gpu4pyscf/df/int3c2e.py

```diff
@@ -1084,7 +1084,8 @@ def get_dh1e(mol, dm0):
     charges = mol.atom_charges()
     fakemol = gto.fakemol_for_charges(coords)
     intopt = VHFOpt(mol, fakemol, 'int2e')
-    intopt.build(1e-14, diag_block_with_triu=True, aosym=False, group_size=BLKSIZE, group_size_aux=BLKSIZE)
+    intopt.build(1e-14, diag_block_with_triu=True, aosym=False, group_size=256, group_size_aux=256)
```

Comment:
Please use the global variable 'BLKSIZE', avoid the hardcode constant.

Reply:
So is this line working now? If I am to adjust the size for HPC, how can I do to prevent exceeding memory usage? Or is it done automatically?

Reply:
Yes, BLKSIZE is defined as 256 on the top of the file.